packages/js/client/src/__tests__/e2e/test-cases.ts

```diff
@@ -473,6 +473,28 @@ export const runJsonTypeTest = async (client: PolywrapClient, uri: string) => {
 
   const methodJSONResult = JSON.stringify(json);
   expect(methodJSONResponse.data).toEqual(methodJSONResult);
+
+  if (testReserved) {
```

Comment:
If this is added, then the `wasm-rs` file should be edited to pass `true` for this arg when calling the `runJsonTypeTest` function